src/RestServer/Controllers/v1/WalletController.cs

```diff
@@ -0,0 +1,646 @@
+        /// <param name="scriptHash" example="0xed7cc6f5f2dd842d384f254bc0c2d58fb69a4761">ScriptHash of the wallet address.</param>
+        /// <returns>A export key object.</returns>
+        /// <response code="200">Successfully exported the key.</response>
+        /// <response code="400">An error occurred. See Response for details.</response>
+        [HttpGet("{session:required}/export/{address:required}", Name = "WalletExportKeysByAddressOrScripthash")]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(WalletExportKeyModel))]
+        public IActionResult WalletExportKey(
+            [FromRoute(Name = "session")]
+            Guid sessionId,
+            [FromRoute(Name = "address")]
+            UInt160 scriptHash)
+        {
+            if (WalletSessions.ContainsKey(sessionId) == false)
+                throw new KeyNotFoundException(sessionId.ToString("n"));
+            var session = WalletSessions[sessionId];
+            session.ResetExpiration();
+            var wallet = session.Wallet;
+            var account = wallet.GetAccount(scriptHash);
+            if (account == null)
+                throw new WalletException($"Account {scriptHash} doesn\'t exist.");
+            var key = account.GetKey();
+            return Ok(new WalletExportKeyModel
+            {
+                ScriptHash = account.ScriptHash,
+                Address = account.Address,
+                Wif = key.Export(),
+            });
+        }
+
+        /// <summary>
+        /// Create a new address in the wallet with an optional private key.
+        /// </summary>
+        /// <param name="sessionId" example="066843daf5ce45aba803587780998cdb">Session Id of the open/created wallet.</param>
+        /// <param name="model"></param>
+        /// <returns>Wallet address object.</returns>
+        /// <response code="200">Successfully created address.</response>
+        /// <response code="400">An error occurred. See Response for details.</response>
+        [HttpPost("{session:required}/address/create", Name = "WalletCreateAddress")]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(WalletAddressModel))]
+        public IActionResult WalletCreateNewAddress(
+            [FromRoute(Name = "session")]
+            Guid sessionId,
+            [FromBody]
+            WalletCreateAccountModel model)
+        {
+            if (WalletSessions.ContainsKey(sessionId) == false)
+                throw new KeyNotFoundException(sessionId.ToString("n"));
+            var session = WalletSessions[sessionId];
+            session.ResetExpiration();
+            var wallet = session.Wallet;
+            var account = Wallet.GetPrivateKeyFromWIF(model.Wif) == null || model.Wif.Length == 0 ?
+                wallet.CreateAccount() :
+                wallet.CreateAccount(Wallet.GetPrivateKeyFromWIF(model.Wif));
```

Comment:
```suggestion
            WalletAccount account;
            if (model.Wif == null || model.Wif.Length == 0)
            {
                account = wallet.CreateAccount();
            }
            else
            {
                account = wallet.Import(model.Wif);
            }
```
Sorry, `Wallet.GetPrivateKeyFromWIF()` can't import a null parameter, let's change it like this.

Reply:
Its Ok for these `Controller` actions to throw exceptions. Server will handle them properly and output as `ErrorModel`. `GetPrivateKeyFromWIF` will throw `ArgumentNullException` or `FormatException`.

But we can change it in this situation. Seeing how the error we will get won't indicate what really is going on. And on top of that we want to `CreateAccount`

Reply:
> Its Ok for these `Controller` actions to throw exceptions. Server will handle them properly and output as `ErrorModel`. `GetPrivateKeyFromWIF` will throw `ArgumentNullException` or `FormatException`.
I mean if this situation wif is necessary, we can't create a random address using `wallet.CreateAccount();` :)